index.js

```diff
@@ -503,7 +504,7 @@ class NewRegressions {
           test.file = v;
           break;
         case 'ONE_STREAM':
```

Comment:
this is not handled in the v suite and i think onstream is a bad approach

Reply:
so far it's the only approach that works if a command produces both warning/error messages and normal output, and there is a need to determine whether the output order is correct